src/main/tut/index.html

```diff
@@ -296,6 +332,12 @@
 implicit val booleanDecoder = cellDecoder(_.toBoolean)
 ```
 
+<!-- COMMENT: earlier in the talk you mentioned the pitfalls of implicit scope,
```

Comment:
Good point, but I don't actually think tut lets me declare companion objects. Or does it? Must investigate.
Thanks!